src/resources/EnplugConfiguration.test.js

```diff
@@ -0,0 +1,54 @@
+import chai from 'chai';
+import fetchMock from 'fetch-mock';
+
+import chaiAsPromised from 'chai-as-promised';
+import Client from '../Client';
+import EnplugConfiguration from './EnplugConfiguration';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('When instantiating an enplug configuration based on customer and enplug serial', () => {
+  const client = new Client();
+  const enplugConfiguration = new EnplugConfiguration(client, EnplugConfiguration.makeHref('SYNC', 'SERIAL#11'));
```

Comment:
Is the `#` here intended to be part of a valid serial number or is it indicating a hash in the URL? Is `#` a potential serial number character?

Reply:
no its not, i removed it, as it does seem confusing to use a hash there